src/main/java/org/embulk/input/jira/util/JiraUtil.java

```diff
@@ -61,21 +66,22 @@ public static void validateTaskConfig(final PluginTask task)
         if (isNullOrEmpty(uri)) {
             throw new ConfigException("JIRA API endpoint could not be empty");
         }
-        HttpURLConnection connection = null;
-        try {
-            URL u = new URL(uri);
-            connection = (HttpURLConnection) u.openConnection();
-            connection.setRequestMethod("GET");
-            connection.getResponseCode();
+        try (CloseableHttpClient client = HttpClientBuilder.create()
+                                            .setDefaultRequestConfig(RequestConfig.custom()
+                                                                                .setConnectTimeout(HTTP_TIMEOUT)
+                                                                                .setConnectionRequestTimeout(HTTP_TIMEOUT)
+                                                                                .setSocketTimeout(HTTP_TIMEOUT)
+                                                                                .setCookieSpec(CookieSpecs.STANDARD)
+                                                                                .build())
+                                            .build()) {
+            HttpGet request = new HttpGet(task.getUri());
+            try (CloseableHttpResponse response = client.execute(request)) {
+                response.getStatusLine().getStatusCode();
```

Comment:
Unlike previous code where we have to "probe" for the status code to have the request to execute, getting the status code here is unnecessary, it should had been already executed.
Just:
```
try (Closeable ignored = client.execute(request)) {}
```
will suffice.

Reply:
Done

Reply:
The request is guaranteed to executed without the need of getting status code.
But as this doesn't impact the correctness of this code, so I leave it up to you to decide.